lang/en_gb.lang

```diff
@@ -50,4 +50,4 @@ ycom_perm_all = Access for all users
 
 ycom_this_login_exists_already = This username already exists
 ycom_please_enter_login = Please enter username
-ycom_please_enter_email = Please enter e-mail address
```

Comment:
Jain. Traduko nutzt die de_de Datei als "Template" für alle Sprachen (sodass Leerzeilen an den gleichen Stellen sind etc.). Da in de_de auch keine Newline am Ende ist, wurde sie von traduko hier auch entfernt.